packages/postgress/src/MessagePickupRepositoryPg.ts

```diff
@@ -0,0 +1,616 @@
+import {
+  AddMessageOptions,
+  Agent,
+  EncryptedMessage,
+  GetAvailableMessageCountOptions,
+  injectable,
+  Logger,
+  MessagePickupEventTypes,
+  MessagePickupLiveSessionRemovedEvent,
+  MessagePickupLiveSessionSavedEvent,
+  MessagePickupRepository,
+  QueuedMessage,
+  RemoveMessagesOptions,
+  TakeFromQueueOptions,
+} from '@credo-ts/core'
+import { Pool, Client } from 'pg'
+import PGPubsub from 'pg-pubsub'
+import * as os from 'os'
+import {
+  MessageState,
+  POSTGRES_PASSWORD,
+  POSTGRES_USER,
+  POSTGRES_HOST,
+  FCM_SERVICE_BASE_URL,
+} from '../config/constants'
+import { createTableMessage, tableNameLive, createTableLive } from '../config/dbCollections'
+import { ConnectionInfo } from './interfaces'
+import { MessagePickupSession } from '@credo-ts/core/build/modules/message-pickup/MessagePickupSession'
+import axios from 'axios'
+
+@injectable()
+export class MessagePickupRepositoryPg implements MessagePickupRepository {
+  private logger?: Logger
+  public messagesCollection?: Pool
+  private agent?: Agent
+  private pubSubInstance?: PGPubsub
+  private dbChannel?: string
+  private dbListener?: boolean
+  private instanceName?: string
```

Comment:
Is this used for anything? If not, just keep it simple and remove it.

Reply:
instanceName is used for debugging purposes to identify which instance is sending and receiving messages. The other variables have been removed.